publication-doi-commons/build.gradle

```diff
@@ -5,7 +5,7 @@ repositories {
 dependencies {
     implementation group: 'com.github.BIBSYSDEV', name: 'nva-datamodel-java', version: project.ext.nvaDatamodelVersion
 
-    api group: 'no.unit.nva', name: 'core', version: project.nvaCommonsVersion
+    api group: 'com.github.bibsysdev', name: 'core', version: project.nvaCommonsVersion
```

Comment:
here it sayss project.nvaCommonVersion, and in another it says project.ext.nvaCommonsVersion.

And in the first build.gradle file, you set nvaCommonsVersion but not in other gradle files. Does it mean that the one place you define the version is global for all modules?

Reply:
yes.